tests/run/i16458.scala

```diff
@@ -0,0 +1,61 @@
+
+object Test {
+  import scala.xml.*
+  def main(args: Array[String]): Unit = {
+    val xml =  <div>FooBar</div><!-- /.modal-content -->
```

Comment:
Replace tests/pos/i16458.scala with tests/run/i16458.scala to make sure we can get expected parse result.